docs/reference/installation/package_managers.cue

```diff
@@ -0,0 +1,43 @@
+package metadata
+
+installation: package_managers: {
+	apt: {
+		title:       "APT"
+		description: installation._interfaces.apt.description
+	}
+
+	dpkg: {
+		title:       "DPKG"
+		description: installation._interfaces.dpkg.description
+	}
+
+	helm: {
+		title:       "Helm"
```

Comment:
How about we reuse the name here?

```suggestion
		title:       installation._interfaces."helm3".name
```

Reply:
This will always be just "Helm". But it could have multiple "interfaces" (helm 3, helm 4, etc.